Cesium.Compiler/stdlib/stdio.h

```diff
@@ -1,2 +1,8 @@
 __cli_import("Cesium.Runtime.StdIoFunctions::PutS")
 void puts(char *s); // TODO[#156]: Change to int
+
+void printf(char* s) {
+    // That's temporary until varargs would not be implemented.
+    // no formatting obviously.
+    puts(s);
+}
```

Comment:
I'm not sure if it is valid for `stdio.h` to define the function body in a header file.

Also, this deserves a TODO comment (I'm creating an issue for each merged `TODO`).

Reply:
Okay. that PR is kitchen sink of different failed attempts to find some place where I can have quick win. I will try to cleanup it which is quite possible, but still it reply on earlier RPs which I send, so I waiting for them to land. I will let you know when this would be ready for your attention.

Reply:
For such cases, you may consider marking a PR as a "draft". This way, you formally let me know that it is not yet ready to merge.

An example of a draft PR is #139.

Reply:
Yeah, I was lazy when I create it, but cannot convert PR to draft after creation. Maybe that's some permission issue, or require some setup?

Reply:
Sorry, I'm not sure what the rules about drafts are, but I've found that I can convert it myself. So I did.

Let me know if you want to un-draft this, and cannot do it yourself.

Reply:
What's your opinion on this `printf` implementation? Should we still merge it being in a header file?

Reply:
I would like to have `Cesium.Runtime` to be something minimal and provide as much implementation in C as possible. Example of functions which maybe written in C is `atoi`/`ioa`, we almost at the point where we can write them in C. Maybe that's require have something like `stdlib.c` which would be added automatically by compiler. That's just example, not something which I think though.

For marketing reasons, I think we should have `printf` even with limitation as serious as it is right now. Technically right now we don't have "Hello World".

Reply:
My whole point is that in C, you cannot write code in headers.

If we want to write C stdlib (how do you distinguish it from `Cesium.Runtime`, by the way?), then we'll have to do it in C files, and link them somehow with the user's program.

For now, this is too much. Let's migrate this to the `Cesium.Runtime`, and create a separate issue to discuss details of implementation parts of the standard library in C.